vault/identity_store_oidc.go

```diff
@@ -1531,7 +1570,7 @@ func (i *IdentityStore) oidcPeriodicFunc(ctx context.Context) {
 				i.Logger().Warn("error expiring OIDC public keys", "err", err)
 			}
 
-			i.oidcCache.Flush(nilNamespace)
+			i.oidcCache.Flush(noNamespace)
```

Comment:
Here (and elsewhere in this PR where you call flush), and in SetDefault below, you've added error returns to the functions but aren't actually checking them here.